config/config-feature-flags.yaml

```diff
@@ -82,3 +82,8 @@ data:
   # This is an experimental feature and thus should still be considered
   # an alpha feature.
   enable-tekton-oci-bundles: "false"
+  # Setting this flag to "true" enables the use of custom tasks from
+  # within pipelines.
+  # This is an experimental feature and thus should still be considered
+  # an alpha feature.
+  enable-custom-tasks: "false"
```

Comment:
Should we call it `enable-custom-run` since it can include `task` and `pipeline` in future?

Reply:
@GregDritschler @ImJasonH ok with `enable-custom-run` as feature-flag name ?

Reply:
"Custom run" sounds funny to me.  The feature is called "custom tasks" not "custom run".  Also the flag does not affect whether Run can be used directly.  It only controls whether pipelines can create runs.

Reply:
I don't have a strong feeling about this. I think that even though the CRD is called `Run`, "custom-task" is more readable as config option name.  And in future we might have the `PipelineTask` custom task :D

Reply:
`custom-tasks` sounds more explicit to me, `custom-runs` feels like a feature flag for some custom *execution* of how task are run and not necessary about Pipeline,Task/Runs